GCC missing?

In my AppVeyor project I get the following error at check time:

```
checking whether package 'mboost' can be installed ...Warning: running command '"d:/R/bin/i386/Rcmd.exe" INSTALL -l "C:/projects/mboost/pkg/mboostPatch/mboost.Rcheck" --build --no-html "C:\projects\mboost\pkg\MBOOST~2\MBOOST~1.RCH\00_PKG~1\mboost"' had status 1
 ERROR
Installation failed.
See 'C:/projects/mboost/pkg/mboostPatch/mboost.Rcheck/00install.out' for details.
```

00install.out shows the following error:
```
* installing *source* package 'mboost' ...
** libs
 
*** arch - i386
gcc -m32 -I"d:/R/include" -DNDEBUG     -I"d:/RCompile/r-compiling/local/local323/include"     -O3 -Wall  -std=gnu99 -mtune=core2 -c mboost.c -o mboost.o
/bin/sh: gcc: command not found
make: *** [mboost.o] Error 127
Warning: running command 'make -f "Makevars" -f "d:/R/etc/i386/Makeconf" -f "d:/R/share/make/winshlib.mk" SHLIB="mboost.dll" OBJECTS="mboost.o"' had status 2
ERROR: compilation failed for package 'mboost'
```

I am not exactly sure why this happens. Is gcc really missing or just expected in the wrong place? And how can I solve this?

--- a/scripts/appveyor-tool.ps1
+++ b/scripts/appveyor-tool.ps1
@@ -74,8 +74,9 @@ Function Bootstrap {
   Progress "Setting PATH"
   $env:PATH = $RDrive + '\R\bin\i386;' + 'C:\MinGW\msys\1.0\bin;' + $env:PATH
 
-  if ( Test-Path variable:global:Rtools ) {
+  #if ( Test-Path variable:global:Rtools ) {
 
+  if ( Test-Path "/**/src" ) {
   Progress "Downloading Rtools.vhd"
   bash -c 'curl -s -L https://rportable.blob.core.windows.net/r-portable/master/Rtools.vhd.gz | gunzip -c > ../Rtools.vhd'
 

--- a/scripts/appveyor-tool.ps1
+++ b/scripts/appveyor-tool.ps1
@@ -74,7 +74,7 @@ Function Bootstrap {
   Progress "Setting PATH"
   $env:PATH = $RDrive + '\R\bin\i386;' + 'C:\MinGW\msys\1.0\bin;' + $env:PATH
 
-  if ( Test-Path "src" -or Test-Path variable:global:Rtools) {
+  if ( Test-Path "src" -Or Test-Path variable:global:Rtools) {
 
   Progress "Downloading Rtools.vhd"
   bash -c 'curl -s -L https://rportable.blob.core.windows.net/r-portable/master/Rtools.vhd.gz | gunzip -c > ../Rtools.vhd'

--- a/scripts/appveyor-tool.ps1
+++ b/scripts/appveyor-tool.ps1
@@ -74,7 +74,7 @@ Function Bootstrap {
   Progress "Setting PATH"
   $env:PATH = $RDrive + '\R\bin\i386;' + 'C:\MinGW\msys\1.0\bin;' + $env:PATH
 
-  if ( Test-Path "src" -Or Test-Path variable:global:Rtools) {
+  if ( Test-Path variable:global:Rtools ) {
 
   Progress "Downloading Rtools.vhd"
   bash -c 'curl -s -L https://rportable.blob.core.windows.net/r-portable/master/Rtools.vhd.gz | gunzip -c > ../Rtools.vhd'

--- a/scripts/appveyor-tool.ps1
+++ b/scripts/appveyor-tool.ps1
@@ -74,7 +74,7 @@ Function Bootstrap {
   Progress "Setting PATH"
   $env:PATH = $RDrive + '\R\bin\i386;' + 'C:\MinGW\msys\1.0\bin;' + $env:PATH
 
-  if ( Test-Path "src" ) {
+  if ( Test-Path "src" -or Test-Path variable:global:Rtools) {
 
   Progress "Downloading Rtools.vhd"
   bash -c 'curl -s -L https://rportable.blob.core.windows.net/r-portable/master/Rtools.vhd.gz | gunzip -c > ../Rtools.vhd'

--- a/scripts/appveyor-tool.ps1
+++ b/scripts/appveyor-tool.ps1
@@ -74,8 +74,6 @@ Function Bootstrap {
   Progress "Setting PATH"
   $env:PATH = $RDrive + '\R\bin\i386;' + 'C:\MinGW\msys\1.0\bin;' + $env:PATH
 
-  #if ( Test-Path variable:global:Rtools ) {
-
   if ( Test-Path "/**/src" ) {
   Progress "Downloading Rtools.vhd"
   bash -c 'curl -s -L https://rportable.blob.core.windows.net/r-portable/master/Rtools.vhd.gz | gunzip -c > ../Rtools.vhd'

